### PR TITLE
backend connection to db

### PR DESCRIPTION
**Pull request by printBart**
connection for node.js

### CODE REVIEW

**Review by mikemd403 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -4,8 +4,7 @@
 web/.idea
 
 # dependencies
-web/node_modules
```
Comment (mikemd403) [inline .gitignore:7]: Should leave this. Don't want a Macadamian situation LOL

File: .gitignore [hunk 2]
```diff
@@ -4,8 +4,7 @@
...
-web/server/node_modules
```
Comment (mikemd403) [inline .gitignore:8]: Leave this too.

**Review by mikemd403 — CHANGES_REQUESTED** [review]
Apply requested changes to .gitignore and do another merge request and i'll push it out.

File: .gitignore (hunk 1 above)
Comment (printBart) [reply .gitignore:7]: I added "node_modules/" on the root gitignore. That should ignore all the directories name "node_module" in subdirectories.

File: .gitignore (hunk 2 above)
Comment (printBart) [reply .gitignore:8]: same here

**Review by mikemd403 — APPROVED** [review]

**Review by mikemd833 — COMMENTED** [review]
Approved

**Pull request merged**